Examples/simple.json

```diff
@@ -1,59 +1,63 @@
-[ 'qc-json-1.0.0',
-{
-    'raw_output': 'Output storing was not requested.',
-    'molecule': {
-        'geometry': {
-            'val': [[0, 0, 0], [0, 0, 1]],
-            'units': 'angstrom'
+[
+    "qc-json-1.0.0",
+    {
+        "provenance": {
+            "creator": "My QM Program",
+            "version": "1.1rc1",
+            "database": "https://pqr.pitt.edu/mol/HEFNNWSXXWATRW-JTQLQIEISA-N"
         },
-        'atoms': ['He', 'He']
-    },
-    'driver': 'energy',
-    'method': {
-        'expression': 'SCF',
-        'basis': 'sto-3g'
-    },
-    'variables': {
-        'SCF N ITERS': 2.0,
-        'SCF DIPOLE Y': {
-            'val': 0.0,
-            'units': 'hartree'
+        "molecule": {
+            "geometry": [
+                [
+                    0,
+                    0,
+                    0
+                ],
+                [
+                    0,
+                    0,
+                    1
+                ]
+            ],
+            "atoms": [
+                "He",
+                "He"
+            ]
         },
-        'SCF TOTAL ENERGY': {
-            'val': -5.433191881443323,
-            'units': 'hartree'
+        "driver": "energy",
+        "keywords": {
+            "method": "SCF",
+            "basis": "sto-3g",
+            "_myqc-grid": "mygrid123",
+            "_myqc-guess": "harris"
         },
-        'SCF TWO-ELECTRON ENERGY': {
-            'val': 4.124089347186247,
-            'units': 'hartree'
+        "properties": {
+            "SCF N ITERS": 2.0,
```

Comment:
Do we want to use spaces in keys, or camelCase, or snake_case?

Reply:
I vote for snake_case.